<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sf::AcquisitionPropertyPage</class>
 <widget class="sf::PropertyPage" name="sf::AcquisitionPropertyPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AcquisitionPropertyPage</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblSwitchId">
     <property name="toolTip">
      <string>Id for switiching between multiple acquisition devices.</string>
     </property>
     <property name="text">
      <string>&amp;Switch Id</string>
     </property>
     <property name="buddy">
      <cstring>iieSwitchId</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="sf::InformationIdEdit" name="iieSwitchId"/>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbDeviceUt"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblDeviceUt">
     <property name="toolTip">
      <string>Id for switching between multiple acquisition devices.</string>
     </property>
     <property name="text">
      <string>Device &amp;UT</string>
     </property>
     <property name="buddy">
      <cstring>iieSwitchId</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblDeviceEt">
     <property name="toolTip">
      <string>Id for switiching between multiple acquisition devices.</string>
     </property>
     <property name="text">
      <string>Device &amp;ET</string>
     </property>
     <property name="buddy">
      <cstring>iieSwitchId</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cbDeviceEt"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>sf::InformationIdEdit</class>
   <extends>QLineEdit</extends>
   <header>gii/qt/InformationIdEdit.h</header>
  </customwidget>
  <customwidget>
   <class>sf::PropertyPage</class>
   <extends>QWidget</extends>
   <header>misc/qt/PropertyPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
